api/src/main/java/org/apache/iceberg/encryption/EncryptedOutputFile.java

```diff
@@ -37,4 +37,9 @@ public interface EncryptedOutputFile {
    * #encryptingOutputFile()}.
    */
   EncryptionKeyMetadata keyMetadata();
+
+  /** Underlying output file for native encryption. */
+  default OutputFile rawOutputFile() {
+    return null;
```

Comment:
Should this throw `UnsupportedOperationException`? That seems like a good idea to me so that we don't get NPE when trying to use this with Parquet.